pkg/libs/serializer/serializer.go

```diff
@@ -46,10 +46,12 @@ func (a *Serializer) StringWithUInt16Len(s string) error {
 
 // StringWithUInt32Len writes to the buffer `buf` four bytes of the string's `s` length followed with the bytes of string itself.
 func (a *Serializer) StringWithUInt32Len(s string) error {
-	if len(s) > math.MaxUint32 {
-		return errors.Errorf("too long string, expected max %d, found %d", math.MaxUint32, len(s))
+	len_s_u32 := uint32(len(s))
```

Comment:
Please, rename `len_s_u32` and `max_u32` variables using camelCase notation.

Reply:
now `lenStrU32` and `maxU32`, respectively